.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
 node_modules/
+/.idea
```

Comment:
IDE specific folders should be in your system's global `.gitignore` not in the project `.gitignore`. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore